### PR TITLE
Add solid water data.

### PR DESCRIPTION
**Pull request by nbassler**

### CODE REVIEW

**Comment by nbassler** [comment]
unless anyone has other data, this could be OK start

File: data/resources/materials/README.md [hunk 1]
```diff
@@ -40,9 +56,10 @@ https://physics.nist.gov/cgi-bin/Star/compos.pl?refer=ap&matno=104
 Density: 1.20479E-03 g/cm3
```
Comment (grzanka) [inline data/resources/materials/README.md:56]: Mention that this is density at 20*C

**Review by grzanka — APPROVED** [review]
LGTM apart from minor comment (mention at which temperature the density is defined)

**Review by grzanka — APPROVED** [review]
LGTM

**Pull request merged**